File: GridView/EFCodeFirstInHierarchy/RadGridViewEFCodeFirstInHierarchy/RadGridViewEFCodeFirst.Data/Migrations/Configuration.cs
namespace RadGridViewEFCodeFirst.Data.Migrations
{
    using System;
    using System.Data.Entity.Migrations;
    using System.Linq;

    internal sealed class Configuration : DbMigrationsConfiguration<RadGridViewEFCodeFirst.Data.RadGridViewEFCodeFirstContext>
    {
        public Configuration()
        {
            this.AutomaticMigrationsEnabled = true;
            this.AutomaticMigrationDataLossAllowed = true;
            this.ContextKey = "RadGridViewEFCodeFirst.Data.RadGridViewEFCodeFirstContext";
        }

        protected override void Seed(RadGridViewEFCodeFirst.Data.RadGridViewEFCodeFirstContext context)
        {
        }
    }
}

File: GridView/EFCodeFirstInHierarchy/RadGridViewEFCodeFirstInHierarchyVB/RadGridViewEFCodeFirst.Data/Migrations/Configuration.vb
Imports System
Imports System.Data.Entity
Imports System.Data.Entity.Migrations
Imports System.Linq

Namespace Migrations

    Friend NotInheritable Class Configuration 
        Inherits DbMigrationsConfiguration(Of RadGridViewEFCodeFirstContext)

        Public Sub New()
            AutomaticMigrationsEnabled = True
            Me.AutomaticMigrationDataLossAllowed = True
            Me.ContextKey = "RadGridViewEFCodeFirst.Data.RadGridViewEFCodeFirstContext"
        End Sub

        Protected Overrides Sub Seed(context As RadGridViewEFCodeFirstContext)

        End Sub

    End Class

End Namespace
